proprietary/tokens/src/common/ams/typography.tokens.json

```diff
@@ -0,0 +1,61 @@
+{
+  "ams": {
+    "common": {
+      "typography": {
+        "body-text": {
+          "font-family": { "value": "{ams.brand.typography.font-family.body-text}" },
+          "font-size": { "value": "{ams.brand.typography.font-size.md}" },
+          "font-weight": { "value": "{ams.brand.typography.font-weight.normal}" },
+          "line-height": { "value": "{ams.brand.typography.line-height.5xl}" },
+          "sm": {
+            "font-size": { "value": "{ams.brand.typography.font-size.sm}" },
+            "line-height": { "value": "{ams.brand.typography.line-height.2xl}" }
+          },
+          "lg": {
+            "font-size": { "value": "{ams.brand.typography.font-size.lg}" },
+            "line-height": { "value": "{ams.brand.typography.line-height.2xl}" }
+          },
+          "xl": {
+            "font-size": { "value": "{ams.brand.typography.font-size.xl}" },
+            "line-height": { "value": "{ams.brand.typography.line-height.md}" }
+          }
+        },
+        "bold": {
+          "font-weight": { "value": "{ams.brand.typography.font-weight.bold}" }
+        },
+        "headings": {
+          "font-family": { "value": "{ams.brand.typography.font-family.headings}" },
```

Comment:
Having two separate tokens for the `font-family` of headings and body text, while we use the same font for both, allows the community to use two different typefaces.